src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('span.population');
+const averagePopulation = document.querySelector('span.average-population');
+const totalPopulation = document.querySelector('span.total-population');
+const numbers = Array.from(populations).map((span) => {
+  const row = span.textContent;
+
+  return +row.split(',').join('');
+});
```

Comment:
Consider adding a check to ensure that the text content of each span is a valid number before converting it. This will help prevent potential errors if the content is not as expected.